src/main/java/com/exactpro/th2/common/schema/grpc/router/impl/DefaultGrpcRouter.java

```diff
@@ -61,7 +66,32 @@
      * @throws ClassNotFoundException if matching the service class to protobuf stub has failed
      */
     public <T> T getService(@NotNull Class<T> cls) throws ClassNotFoundException {
-        return getProxyService(cls);
+        var implementations = ServiceLoader.load(Objects.requireNonNull(cls, "Services class can not be null"))
```

Comment:
Please, specify the result type explicitly. It is not obvious that the result collection has the following type
```java
List<ServiceLoader.Provider<T>>
```